internal/ghinstance/host.go

```diff
@@ -73,6 +73,13 @@ func GraphQLEndpoint(hostname string) string {
 	if isGarage(hostname) {
 		return fmt.Sprintf("https://%s/api/graphql", hostname)
 	}
+	// Once we change Tenancy to no longer be treated as Enterprise, this
+	// conditional can be removed as the flow will fall through to the bottom.
+	// However, we can't do that until we've investigated all places in which
+	// Tenancy is currently treated as Enterprise.
+	if IsTenancy(hostname) {
+		return fmt.Sprintf("https://api.%s/graphql", hostname)
+	}
```

Comment:
Do we have a separate issue to follow up on this?  Any specific places offhand you know are a problem?